setup.py

```diff
@@ -39,28 +25,28 @@ def load_requirements(path_dir=PATH_ROOT, comment_char='#'):
 # engineer specific practices
 setup(
     name='pytorch-lightning-bolts',
-    version=pytorch_lightning_bolts.__version__,
-    description=pytorch_lightning_bolts.__docs__,
-    author=pytorch_lightning_bolts.__author__,
-    author_email=pytorch_lightning_bolts.__author_email__,
-    url=pytorch_lightning_bolts.__homepage__,
+    version=pl_bolts.__version__,
+    description=pl_bolts.__docs__,
+    author=pl_bolts.__author__,
+    author_email=pl_bolts.__author_email__,
+    url=pl_bolts.__homepage__,
     download_url='https://github.com/PyTorchLightning/pytorch-lightning-bolts',
-    license=pytorch_lightning_bolts.__license__,
-    packages=find_packages(exclude=['tests']),
+    license=pl_bolts.__license__,
+    packages=find_packages(exclude=['tests', 'docs']),
 
-    long_description=open('README.md', encoding='utf-8').read(),
+    long_description=pl_bolts.__long_doc__,
     long_description_content_type='text/markdown',
     include_package_data=True,
     zip_safe=False,
 
     keywords=['deep learning', 'pytorch', 'AI'],
     python_requires='>=3.6',
     setup_requires=[],
-    install_requires=load_requirements(PATH_ROOT),
+    install_requires=['pytorch-lightning>=0.7.1'],
```

Comment:
Either add torch vision as requirement or make imports optional in the code. Otherwise the in installation will fail, when you don't install torch vision manually